test.sh

```diff
@@ -236,6 +238,17 @@ clean_up()
     fi
 }
 
+convert_notebooks()
+{
+    echo "testing notebooks"
+    for notebook in `grep ipynb docs/tutorials.rst | sed -e 's/^[ \t]*//'`
+    do
+        jupyter nbconvert --to notebook --execute "docs/$notebook"
+        check_errs $?
+        rm docs/*.nbconvert.ipynb
```

Comment:
1. When the nbconvert errors out, then the `rm` command doesn't get executed
2. Since nbconvert is generating files, we should add `*.nbconvert.ipynb` to the `.gitignore` file

Reply:
> 1. When the nbconvert errors out, then the `rm` command doesn't get executed
> 
> 2. Since nbconvert is generating files, we should add `*.nbconvert.ipynb` to the `.gitignore` file 

Very good point! I noticed the rm command not being executed when an error occurs but I forgot to think and address it! 

Since rm command is not effective when there is an error, I think we should just remote it and handle it in gitignore. Is that correct?

Reply:
It is still good to remove the files when possible but, in this case, I would move the `rm` command into the `clean_up` function. When there is failure, one would just need to clean things up manually

Separately, regardless, we should add the file type to `.gitignore`